salt/modules/zypper.py

```diff
@@ -511,6 +511,7 @@ def refresh_db():
 def install(name=None,
             refresh=False,
             fromrepo=None,
+            skip_verify=False,
```

Comment:
Can you please move the to the end of the list, before **kwargs? It should align with the argspec from pkgng.py. Thanks.